pkg/xen-tools/xen-start

```diff
@@ -90,7 +92,7 @@ ID=$(domID "$1")
 # finally unpause the domain
 xl unpause "$ID" || bail "xl unpause failed"
 
-# now star polling for domain status (11 sec interval) in the background
+# now star polling for domain status in the background
```

Comment:
s/star/start/

Might make sense to specify that the domID function has  a delay which prevents this from being a busy-loop.

Reply:
Or improve understandability by moving the sleep from the domID function to its caller(s)??